test/services/katello/pulp/rpm_test.rb

```diff
@@ -2,14 +2,14 @@
 require 'support/pulp/repository_support'
 
 module Katello
-  class GluePulpRpmTestBase < ActiveSupport::TestCase
+  class ServicesPulpRpmTestBase < ActiveSupport::TestCase
```

Comment:
What about wrapping this in module declaration for Services and Pulp vs this long test name that encompasses various layers?